components/page-footer.tsx

```diff
@@ -28,53 +75,49 @@ export function PageFooter({
             asChild
           >
             <Link href="/">
-              <span className="sr-only">Brand logo for Triskacode</span>
+              <span
+                className="sr-only"
+                aria-label="Link to Triskacode's homepage"
+              >
+                Brand logo for Triskacode
+              </span>
               _triskacode
             </Link>
           </Button>
           <div className="flex items-center justify-start gap-2">
-            <Button variant="outline" size="icon" asChild>
+            <Button
+              asChild
+              variant="outline"
+              size="icon"
+              aria-label="Link to Triskacode's GitHub profile"
+            >
               <Link href="https://github.com/triskacode" target="_blank">
                 <span className="sr-only">GitHub profile</span>
                 <Icon.GitHub className="h-4 w-4" />
               </Link>
             </Button>
           </div>
         </div>
-        <div className="flex flex-col gap-4">
-          <div className="text-sm font-semibold">
-            <p>Resources</p>
+        {footerLinks.map(({ title, links }) => (
+          <div key={title} className="flex flex-col gap-4 text-sm">
+            <div className="font-semibold">{title}</div>
+            <ul className="flex flex-col gap-2">
+              {links.map(({ text, href, target }) => (
+                <li key={text}>
+                  <Button
+                    variant="link"
+                    className="justify-start px-0 text-sm hover:no-underline"
+                    asChild
+                  >
+                    <Link href={href} target={target}>
+                      {text}
+                    </Link>
+                  </Button>
+                </li>
```

Comment:
Remove unnecesary `Button` component
```suggestion
                 <li key={text}>
                    <Link href={href} target={target}>
                      {text}
                    </Link>
                </li>
```